CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.2.2] - 2020-11-21
+
+- Add support for unit structs serialization / deserialization.
```

Comment:
Let's collect those under the `## [Unreleased]` until there is a release. The date should match the date of publishing